<ui version="4.0" >
 <class>ExtractingProgressWidgetUI</class>
 <widget class="QWidget" name="ExtractingProgressWidgetUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label_header" >
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>&lt;h2>Artist - Album&lt;/h2></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KListWidget" name="klistwidget" >
     <property name="minimumSize" >
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label_extracting" >
       <property name="text" >
        <string>Ripping Track 8 / 12</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KSqueezedTextLabel" name="label_track" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="textElideMode" >
        <enum>Qt::ElideLeft</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_speed_extracting" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Speed: 0x</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar_extracting" >
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_overall" >
     <property name="text" >
      <string>Overall Ripping Progress</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar_overall" >
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label_encoding" >
       <property name="text" >
        <string>Encoding Track 8 / 12</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KSqueezedTextLabel" name="filename" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text" >
        <string>KSqueezedTextLabel</string>
       </property>
       <property name="textElideMode" >
        <enum>Qt::ElideLeft</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_speed_encoding" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Speed: 0x</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar_encoding" >
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KSqueezedTextLabel</class>
   <extends>QLabel</extends>
   <header>ksqueezedtextlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
